src/components/IssueLabels.js

```diff
@@ -0,0 +1,29 @@
+import { array, number } from 'prop-types'
+import React from 'react'
+import { spacing } from '../theme'
+import Flex from './Flex'
+import IssueLabel from './IssueLabel'
+
+function IssueLabels({ labels, totalCount }) {
+  const remainingLabels = totalCount - labels.length
+
+  return (
+    totalCount > 0 && (
+      <Flex css={{ flexWrap: 'wrap', marginTop: spacing[0] }}>
+        {labels.map((label, index) => (
+          <IssueLabel key={index} color={`#${label.color}`}>
```

Comment:
It's generally bad practice to use `index` as the key. We could use `label.name`.